content/p2p/fast-feedback/_index.md

```diff
@@ -10,8 +10,24 @@ The aim of this pipeline shape is to provide fast feedback on the provided code.
 We've defined a pipeline using github actions [here](https://github.com/coreeng/p2p).
 
 What we propose looks like this:
-{{< figure src="/images/p2p/fast-feedback.png" title="Fast Feedback" >}}
-
+{{< mermaid >}}
+sequenceDiagram
+actor User
+User->>+Git: push to branch
+Git->>+Git: build image
+Git->>+Dev Registry: Push image to '/test'    
+Git->>+Dev k8s: Deploy to Functional + Run tests
+Git->>+Dev k8s: Deploy to NFT + Run tests
+Git->>+Dev k8s: Deploy to Integration + Run tests
+alt branch is main
+Git->>+Dev Registry: Promote image from '/test' to '/extended-test'
```

Comment:
Isn't this happening in a feature branch?

Reply:
Reference app example:
https://github.com/coreeng/idp-reference-app-go/blob/25690d835bedaf6212dad680c01495f187cb43e6/.github/workflows/fast-feedback.yaml#L8

Looks like we both are right :)
From my perspective feature branch is more suited here just because Fast-Feedback is meant to be run quickly on PRs. But I don't insist here

Reply:
ah, actually it is correct as it is, it is wrapped into alt, will clarify now